.circleci/config.yml

```diff
@@ -331,6 +331,47 @@ jobs:
                   docker login --username  $DOCKER_HUB_USER --password $DOCKER_HUB_PW
                   docker push prefecthq/prefect:latest
 
+  build_master_docker_image:
+    docker:
+      - image: docker
+    parameters:
+      python_version:
+        type: string
+    environment:
+      PYTHON_VERSION: << parameters.python_version >>
+    steps:
+      - checkout
+      - run:
+          name: Master branch check
```

Comment:
technically this isn't necessary since there isn't a combined tag/branch filter (also doesn't hurt to leave it)

Reply:
You're right, I'm going to remove it since it doesn't seem necessary to check the same thing twice considering the circle filter already does it